website_product_filters/models/product.py

```diff
@@ -80,11 +80,10 @@ class ProductCategory(models.Model):
         "product_template_id", readonly=True)
     total_tree_products = fields.Integer("Total Subcategory Prods",
                                          compute="_get_product_count",
-                                         store=True,)
+                                         store=False,)
```

Comment:
@oscarolar 
I don't have a full context of this change, but the `store` param affect directly the instance migrated you should ask to @josemoralesp